src/toast/toast/custom-toast.tsx

```diff
@@ -22,10 +23,19 @@ interface Props {
   hide: EmptyFn;
   toastType: ToastTypeEnum;
   operationHash?: string;
+  isEstimationError?: boolean;
```

Comment:
maybe rename to `isCopyButtonVisible` to make this prop reusable in other places